compiler/src/dotty/tools/dotc/Compiler.scala

```diff
@@ -109,6 +109,9 @@ class Compiler {
            new LabelDefs),          // Converts calls to labels to jumps
       List(new GenBCode)            // Generate JVM bytecode
     )
+    assert(allPhases.last.last.id <= Periods.MaxPossiblePhaseId)
```

Comment:
\+ are phase ids set here at all? isn't `allPhases.last.last.id` not yet initialized here?